.github/workflows/release-publish.yml

```diff
@@ -27,9 +27,12 @@ jobs:
           registry: ghcr.io
           username: ${{ github.repository_owner }}
           password: ${{ secrets.GITHUB_TOKEN }}
+      - name: make empty ts client
+        run: mkdir -p ./gen-ts-client
```

Comment:
I still wish the build was possible without this, but alas

Allthough maybe we should add a `docker` target to the makefile that people can use locally, as not doing this would lead to some potentially confusing errors

Reply:
Yes, and we could include the necessary arguments, as `docker build .` doesn't work out of the box. I usually run with `DOCKER_BUILDKIT=1 docker build . --progress plain --platform linux/amd64 -t authentik:test` on a Linux machine.

Reply:
TIL about `--progress plain`